experimental/Schemes/Auxiliary.jl

```diff
@@ -0,0 +1,47 @@
+@Markdown.doc """
+    pullback(f::CoveredSchemeMorphism)
+
+Return a function `phi` which takes any reasonable 
+argument `v` associated to the `codomain` of `f` and 
+produces ``f*(v)``.
+"""
+function pullback(f::CoveredSchemeMorphism)
+  X = domain(f)
+  Y = codomain(f)
+
+  function pullback_func(II::IdealSheaf)
+    scheme(II) === Y || error("ideal sheaf is not defined on the codomain of the function")
+    phi = covering_morphism(f)
+    ID = IdDict{AbsSpec, Ideal}()
+    for U in patches(domain(phi))
+      f_U = phi[U]
+      V = codomain(f_U)
+      pbf = pullback(f_U)
+      ID[U] = ideal(OO(U), pbf.(gens(II(V))))
+    end
+    return IdealSheaf(X, ID, check=false) # TODO: Set to false eventually.
+  end
```

Comment:
For $f: X \to Y$  and $\mathcal{I}_T \leq \mathcal{O}_X$ this looks like
$f^{-1} \mathcal{F} \cdot \mathcal{O}_X$ to me which is in general not the same as the pullback $f^* \mathcal{I}_T$ (which in general fails to be an ideal sheaf).
See
http://math.stanford.edu/~vakil/216blog/FOAGnov1817public.pdf
16.3.9 for a nice explanation.

Reply:
But this is about the difference of the pullback of `I` seen as a module vs. `I` seen as an ideal. I guess in the latter case we really want $f^{-1}I\cdot \mathscr O_X$, don't we?

Reply:
Yes, I think you are doing the right thing. Though I am not so sure about the name "pullback" . 
At least it needs to be documented individually I think.

Reply:
I added a note in the docstring of idealsheaves. Making an extra docstring for the pullback didn't seem to be the right thing, since the function is too generic.